cartridges/int_algolia_controllers/cartridge/static/default/js/algolia/insights-config.js

```diff
@@ -1,11 +1,23 @@
 /* global aa */
-
+/**
+ * Configures Insights
+ * @param {string} appId Application ID
+ * @param {string} searchApiKey Search API Key
+ */
 function enableInsights(appId, searchApiKey) {
     window.aa('init', {
         appId: appId,
-        apiKey: searchApiKey
+        apiKey: searchApiKey,
+        // the default value was changed to false starting with SearchInsights v2
+        // this means that an anonymous user token will no longer be generated and saved for the session automatically
+        // this will generate 422 errors in the Algolia Events Debugger if useCookie is false and a user token was not specified explicitly
+        // please see the documentation for more details
```

Comment:
You could put a link to the doc actually

Reply:
I think you forgot to push the commit

Reply:
Realized it right after merging the PR -- cherry-picked it onto APEX-721 so it is now on develop.